packages/nuejs.org/blog/tailwind-vs-semantic-css/index.md

```diff
@@ -93,7 +92,7 @@ The two metrics that measure page rendering speed are [first contentful paint](/
 
 
 
-Please compare [Tailwind metrics](//pagespeed.web.dev/analysis/https-spotlight-tailwindui-com/cqtnf4xxoy?form_factor=mobile) with [Semantic CSS metcis ](//pagespeed.web.dev/analysis/https-nuejs-org-spotlight/6nnhwwnz8b?form_factor=mobile).
+Please compare [Tailwind metrics](//pagespeed.web.dev/analysis/https-spotlight-tailwindui-com/cqtnf4xxoy?form_factor=mobile) with [Semantic CSS metrics](//pagespeed.web.dev/analysis/https-nuejs-org-spotlight/6nnhwwnz8b?form_factor=mobile).
```

Comment:
These links do not work anymore, and the page-speeds are negligibly different:
Nue: https://pagespeed.web.dev/analysis/https-nuejs-org-spotlight/i0itoo400b?form_factor=mobile
Tailwind: https://pagespeed.web.dev/analysis/https-spotlight-tailwindui-com/98p017y0jm?form_factor=mobile

Previous comment: https://github.com/nuejs/nue/commit/89aaa518494962c9bdf1f83e67f7c5a7d5b2eb4e#r144877042